### PR TITLE
ADD: external marketplace link for ios

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

**Comment by Overtorment** [comment]
@marcosrdz  can u  pls take it from here? I need this icon to align nicely, plus I want it to be easy to add another button (button should stack in a row), for example one button will be LappBrowser the other one Marketplace, third one is Tools

image

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ https://appetize.io/app/86ft820u48ccj6j6drqq6nq4am

(posted automatically)

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ https://appetize.io/app/w6z3tdt4uk5wwujt5vg8wkxu0c

(posted automatically)

**Comment by marcosrdz** [comment]
Done.

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ https://appetize.io/app/dx55d1xg9x95eh870761jjwtgg

(posted automatically)

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ https://appetize.io/app/p019jxfvk8vrupc9waw7wfe8nm

(posted automatically)

**Comment by Overtorment** [comment]
Not great not terrible

image

File: screen/wallets/transactions.js [hunk 1]
```diff
@@ -85,6 +85,9 @@ export default class WalletTransactions extends Component {
 
   componentDidMount() {
     // nop
+    this.state.wallet.getBalance = () => {
```
Comment (Overtorment) [inline screen/wallets/transactions.js]: I dont get it. whats this?

Comment (Overtorment) [reply screen/wallets/transactions.js]: is it debug code..?

**Comment by Overtorment** [comment]
And how do I add another button for IOS? I need "marketplace" that opens external marketplace, and "LappBrowser" that opens our browser screen

**Comment by Overtorment** [comment]
Test this build in browser:

(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ https://appetize.io/app/87za0673z49hyv914ydpuyr6m8

(posted automatically)

**Pull request merged**